module_intros/cluster.Cluster.ipynb

```diff
@@ -0,0 +1,336 @@
+{
```

Comment:
<p>I'm not sure that this cell adds anything. I think we could delete it and just show the next one, which illustrates the key point of the tensor.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/freud-examples/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>